client/lib/mixins/analytics/index.js

```diff
@@ -525,6 +525,43 @@ const EVENTS = {
 						email
 					}
 				);
+			},
+
+			pendingAccountLogInClick( { domain, user, severity, isMultipleDomains, section } ) {
+				analytics.ga.recordEvent(
+					'Domain Management',
+					`Clicked "Log in" link in Google Apps pending ToS notice in ${ section }`,
+					'Domain Name',
+					domain
+				);
+
+				analytics.tracks.recordEvent(
+					'calypso_domain_management_google_apps_pending_account_log_in_click',
+					{
+						domain,
```

Comment:
Here it is referred as `domain` and the in the other one, it's `domain_name`. I think the convention is to use `domain_name`.